flotilla/endpoints.go

```diff
@@ -95,6 +97,8 @@ func (ep *endpoints) decodeListRequest(r *http.Request) listRequest {
 
 	lr.limit, _ = strconv.Atoi(ep.getURLParam(params, "limit", "1024"))
 	lr.offset, _ = strconv.Atoi(ep.getURLParam(params, "offset", "0"))
+
+	// @TODO not every entity that can be listed has a group name...
 	lr.sortBy = ep.getURLParam(params, "sort_by", "group_name")
```

Comment:
TODO: should probably add another method called decodeListRequestV7 that doesn't assume every entity has a group name field